### DISCUSSION TITLE
How to download Thumbnails & Subtitles using Seal?

### DISCUSSION THREAD

**Discussion by StylesClashNation** [JunkFood02/Seal#1159] [General]
Hey how do I download thumbnails and subs using this app thanks

**Comment by Marco-9456** [comment]
@DeVonRoy1995 
In bulk and without the videos?

**Reply by StylesClashNation** [reply]
Yes

**Reply by Marco-9456** [reply]
**The app doesn't have an option to do this, but you can use the custom command feature and downloads both at the same time or separately:**

### To download both at the same time:

```python
--skip-download
--write-thumbnail
--sub-format FORMAT
--sub-langs LANGUAGE
--write-subs
```

### To download only the subtitles:

```python
--skip-download
--sub-format FORMAT
--sub-langs LANGUAGE
--write-subs
```

### To download only the thumbnails:

```python
--skip-download
--write-thumbnail
```
